cmd/ak/cmd/triggers/create.go

```diff
@@ -18,9 +18,9 @@ var (
 )
 
 var createCmd = common.StandardCommand(&cobra.Command{
-	Use: `create -n name --call file:func [-p project] -c connection [-E event] [-f filter]
-             create -n name --call file:func [-p project] -s "schedule"
-             create -n name --call file:func [-p project] --webhook
+	Use: `create -n name [--call file:func] [-p project] -c connection [-E event] [-f filter]
+             create -n name [--call file:func] [-p project] -s "schedule"
+             create -n name [--call file:func] [-p project] --webhook
```

Comment:
I guess it's possible to define a trigger without an entry point, but it is undesirable to allow it in the client side.

Reply:
why not? you could still wait for it with `next_event` - that's the idea anyway.